05 Thunk/src/components/color/colorDisplayerContainer.tsx

```diff
@@ -1,5 +1,5 @@
 import { connect } from 'react-redux';
-import { ColorDisplayer } from './colordisplayer';
+import { ColorDisplayer } from './colorDisplayer';
```

Comment:
Thanks for the renames @joanrufe. Did you change the file name?

It does not appear here in PR

Thank you